subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -245,60 +315,74 @@ include::sample[dir="userguide/artifacts/defineRepository/groovy",files="build.g
 include::sample[dir="userguide/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=authenticated-ivy-repo]"]
 ====
 
-[[sec:how-gradle-downloads-deps]]
-== How Gradle downloads dependencies
+See <<#sec:supported_transport_protocols>> for authentication options.
 
-Gradle takes your dependency declarations and repository definitions and attempts to download all of your dependencies by a process called _dependency resolution_. Below is a brief outline of how this process works.
+[[sec:repository-content-filtering]]
+== Repository content filtering
 
-* Given a required dependency, Gradle attempts to resolve the dependency by searching for the module the dependency points at. Each repository is inspected in order. Depending on the type of repository, Gradle looks for metadata files describing the module (`.module`, `.pom` or `ivy.xml` file) or directly for artifact files.
+[NOTE]
+====
+Repository content filtering is an <<feature_lifecycle.adoc#feature_lifecycle,incubating>> feature.
```

Comment:
:x: I think It's not going to be incubating in 6.0.

Reply:
Good point, removing!